tests/test_trade_pipeline.py

```diff
@@ -802,6 +955,17 @@ def test_taken_order_expires(self):
 
         self.assert_order_logs(data["id"])
 
+        trade.get_notifications()
+        # Checks
+        self.assertResponse(trade.response)
```

Comment:
If we use `trade.notification` as the response stored by the get_notification() method, that will make tests more clear :)